Description: autotools: rationalize
 Minor fix: the library is written in BASH, as such it is architecture independent,
 so it must be put in an architecture independent directory folder tree, what is not
 the case for $(libdir).
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2016-02-20

--- a/sbin/Makefile.am
+++ b/sbin/Makefile.am
@@ -1,5 +1,7 @@
 # Process this file with automake to produce Makefile.in
 
+libarchinddir = $(prefix)/lib
+
 scriptsin = \
 	firehol.in \
 	fireqos.in \
@@ -7,7 +9,7 @@
 	update-ipsets.in \
 	vnetbuild.in
 
-inclibdir = $(libdir)/firehol
+inclibdir = $(libarchinddir)/firehol
 
 SUFFIXES = .in
 .in:
@@ -17,7 +19,7 @@
 		-e '/^# Start defaults before configure/,/^# End/d' \
 		-e 's#[$$]prefix_POST#$(prefix)#g' \
 		-e 's#[$$]bindir_POST#$(bindir)#g' \
-		-e 's#[$$]libdir_POST#$(libdir)/firehol#g' \
+		-e 's#[$$]libdir_POST#$(inclibdir)#g' \
 		-e 's#[$$]localstatedir_POST#$(localstatedir)#g' \
 		-e 's#[$$]sysconfdir_POST#$(sysconfdir)#g' \
 		-f commands.sed \
